matchbox_socket/src/webrtc_socket/error.rs

```diff
@@ -22,28 +22,36 @@ pub enum ChannelError {
 /// An error that can occur with WebRTC messaging.
```

Comment:
Thanks for improving the docs here. I think we should do a bit more though. Now that Signaller is exported, we should link it here, and clarify that these are from signaling.
```suggestion
/// An error that can occur with WebRTC signaling. See [Signaller].
```